applications/AdjointFluidApplication/tests/test_MainKratos.py

```diff
@@ -32,6 +32,12 @@ def __init__(self, ProjectParameters):
 
```

Comment:
this is fine, however in a future we need to find a cleverer way of doing this.

i am thinking in the concept of "solution_stages"